### PR TITLE
CM-139: add core tasks fe

### PR DESCRIPTION
**Pull request by jdolkowski**
https://openimis.atlassian.net/browse/CM-139

The PR also addresses topics from: https://github.com/openimis/openimis-fe-tasks_management_js/pull/1

it needs: https://github.com/openimis/openimis-fe-tasks_management_js/pull/2

### CODE REVIEW

File: src/components/BenefitPlanTaskPreviewTable.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      <Table size="small" className={classes.table}>
+        <TableHead>
+          <TableRow>
+            {headers().map((column) => (
+              <TableCell>{formatMessage(intl, 'socialProtection', column)}</TableCell>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {/* <ProgressOrError progress={fetchingTasks} error={errorTasks} /> */}
```
Comment (sniedzielski) [inline src/components/BenefitPlanTaskPreviewTable.js:82]: It this commented line on purpose? Do we need this line? @jdolkowski . If no, please remove it.

File: src/components/BenefitPlanTasksSearcher.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+      itemFormatters={itemFormatters}
+      sorts={sorts}
+      rowsPerPageOptions={ROWS_PER_PAGE_OPTIONS}
+      defaultPageSize={DEFAULT_PAGE_SIZE}
+      defaultOrderBy="source"
+      rowIdentifier={rowIdentifier}
+      onDoubleClick={onDoubleClick}
+      defaultFilters={defaultFilters()}
+      rowDisabled={isRowDisabled}
+      // rowLocked={isRowDisabled}
```
Comment (sniedzielski) [inline src/components/BenefitPlanTasksSearcher.js:157]: Please remove commented line

**Review by sniedzielski — COMMENTED** [review]
At this stage please refer to two my comments. Generally looks ok to me. I haven't found nothing critical for me. Please fix those commented lines of code.

File: src/components/BenefitPlanTaskPreviewTable.js (hunk 1 above)
Comment (olewandowski1) [reply src/components/BenefitPlanTaskPreviewTable.js:82]: I think we can get _fetchingTasks_ and _errorTasks_ from the Redux store using _useSelector_ and then just keep this _ProgresOrError_ component.

**Review by olewandowski1 — COMMENTED** [review]
@jdolkowski Overall LGTM. Nevertheless, I see that we're completely inconsistent with a way of implementing styles and translations. We could talk it through and align if needed.

**Review by sniedzielski — APPROVED** [review]

File: src/components/BenefitPlanTaskPreviewTable.js (hunk 1 above)
Comment (jdolkowski) [reply src/components/BenefitPlanTaskPreviewTable.js]: done

File: src/components/BenefitPlanTasksSearcher.js (hunk 2 above)
Comment (jdolkowski) [reply src/components/BenefitPlanTasksSearcher.js]: done

**Pull request merged**